Applications/SseServer/src/SseServer/Configuration.cs

```diff
@@ -13,6 +13,8 @@ public class Configuration
     [Required]
     public InfrastructureConfiguration Infrastructure { get; set; } = new();
 
+    public SseConfiguration Sse { get; set; } = new();
```

Comment:
I think it would be better if we call the configuration property just like the service that uses it (=> `SseServer` instead of `Sse`)